packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -160,7 +162,7 @@ export const ProposalPreview = () => {
 
           {(proposal.status === 'dormant' || votingRounds.length > 1) && (
             <ProposalStages
-              status={proposal.status}
+              roundStatus={proposal.status}
```

Comment:
I don't think this should be changed. This is the status of the proposal so "status" is correct